src/Fixer/LanguageConstruct/IsNullFixer.php

```diff
@@ -0,0 +1,193 @@
+                    continue;
+                }
+
+                // get rid of the root namespace when it used and check if the inversion operator provided
+                $tokens->removeTrailingWhitespace($prevTokenIndex);
+                $tokens[$prevTokenIndex]->clear();
+            }
+
+            // check if inversion being used, text comparison is due to not existing constant
+            $isInvertedNullCheck = false;
+            if ($tokens[$inversionCandidateIndex]->equals('!')) {
+                $isInvertedNullCheck = true;
+
+                // get rid of inverting for proper transformations
+                $tokens->removeTrailingWhitespace($inversionCandidateIndex);
+                $tokens[$inversionCandidateIndex]->clear();
+            }
+
+            /* before getting rind of `()` around a parameter, ensure it's not assignment/ternary invariant */
+            $referenceEnd = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $matches[1]);
+            $isContainingDangerousConstructs = false;
+            for ($paramTokenIndex = $matches[1]; $paramTokenIndex <= $referenceEnd; ++$paramTokenIndex) {
+                if (in_array($tokens[$paramTokenIndex]->getContent(), array('?', '?:', '='), true)) {
+                    $isContainingDangerousConstructs = true;
+                    break;
+                }
+            }
+
+            if (!$isContainingDangerousConstructs) {
+                // closing parenthesis removed with leading spaces
+                $tokens->removeLeadingWhitespace($referenceEnd);
+                $tokens[$referenceEnd]->clear();
+
+                // opening parenthesis removed with trailing spaces
+                $tokens->removeLeadingWhitespace($matches[1]);
+                $tokens->removeTrailingWhitespace($matches[1]);
+                $tokens[$matches[1]]->clear();
+            }
+
+            // sequence which we'll use as a replacement
+            $replacement = array(
+                new Token(array(T_STRING, 'null')),
+                new Token(array(T_WHITESPACE, ' ')),
+                new Token($isInvertedNullCheck ? array(T_IS_NOT_IDENTICAL, '!==') : array(T_IS_IDENTICAL, '===')),
+                new Token(array(T_WHITESPACE, ' ')),
+            );
+
+            if (true === $this->configuration['use_yoda_style']) {
+                $tokens->overrideRange($isNullIndex, $isNullIndex, $replacement);
+            } else {
+                $replacement = array_reverse($replacement);
+                if ($isContainingDangerousConstructs) {
+                    array_unshift($replacement, new Token(array(')')));
+                }
+
+                $tokens[$isNullIndex]->clear();
+                $tokens->removeTrailingWhitespace($referenceEnd);
+                $tokens->overrideRange($referenceEnd, $referenceEnd, $replacement);
+            }
+
+            // nested is_null's support
```

Comment:
No apostrophe here **(I think???)**. This is a very common mistake in English grammar. The support does not **belong to** is_null, you're just using the **plural form** of is_null, right?